app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Model.kt

```diff
@@ -5,16 +5,17 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 /**
- * @param mid: DB ID for dev use
+ * @param mId: DB ID for dev use
  * @param modelName: User Inputted Name
  * @param artistId: Artist ID from Artist table
  * @param modelUrl: model URL
  */
 @Entity(tableName = "Model")
-data class Model (@PrimaryKey var mid: Int,
+data class Model (@PrimaryKey var mId: Int,
                   @ColumnInfo(name = "modelName")var modelName: String,
                   @ColumnInfo(name = "artistId")var artistId: Int,
-                  @ColumnInfo(name = "modelUrl")var url: String) {
+                  @ColumnInfo(name = "modelUrl")var url: String)
+{
```

Comment:
Moved to curly brace to the next line as all other dto's had it on this line. Moved it for consistency and readability.